src/navigator/NavigationService.ts

```diff
@@ -238,7 +240,9 @@ export function navigateHome(options?: NavigateHomeOptions) {
   setTimeout(() => {
     navigationRef.current?.reset({
       index: 0,
-      routes: [{ name: Screens.DrawerNavigator, params }],
+      routes: getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)
```

Comment:
On Android I am getting the error: `The action 'RESET' with payload {"index":0,"routes":[{"name":"TabNavigator"}]} was not handled by any navigator.` likely form here. When testing the non tab navigator path no error is served. On iOS I have not observed this same issue. Perhaps this should be fixed the follow up ACT-1110.

Reply:
i'm confused by your PR comment and the comment in the code: the code says there's a TODO to handle undefined/falsey, the code seems to handle it by reverting to previous behavior (DrawerNavigator), and you PR comment seems to imply that addresses act-1110 will solve a related (unrelated?) problem?

Reply:
Act-1110 is the feature gate ticket, as this relates to the feature gate not being defined when the navigation service is initialized, we try to initialize Statsig as early as possible, it seems appropriate to lump it into that work.